src/core/surface/call.h

```diff
@@ -152,17 +151,17 @@ void grpc_call_context_set(grpc_call *call, grpc_context_index elem,
 void *grpc_call_context_get(grpc_call *call, grpc_context_index elem);
 
 #define GRPC_CALL_LOG_BATCH(sev, call, ops, nops, tag) \
-  if (grpc_trace_batch) grpc_call_log_batch(sev, call, ops, nops, tag)
+  if (grpc_surface_trace) grpc_call_log_batch(sev, call, ops, nops, tag)
```

Comment:
Making a note here so I don't forget: per offline discussion, surface tracing will be coalesced into API tracing.